hoot-services/src/main/java/hoot/services/controllers/grail/PullApiCommand.java

```diff
@@ -134,10 +133,12 @@ private void getApiData() {
             File outputFile = new File(params.getOutput());
 
             if (responseStream == null) {
-                URL requestUrl = new URL(url);
-                FileUtils.copyURLToFile(requestUrl,outputFile, Integer.parseInt(HootProperties.HTTP_TIMEOUT), Integer.parseInt(HootProperties.HTTP_TIMEOUT));
+                responseStream = GrailResource.getUrlInputStreamWithNullHostnameVerifier(url);
+                FileUtils.copyInputStreamToFile(responseStream, outputFile);
+                responseStream.close();
             } else {
                 FileUtils.copyInputStreamToFile(responseStream, outputFile);
+                responseStream.close();
             }
```

Comment:
seems like we really only need the if to set the responseStream if null, rest can be done outside

Reply:
Great point, want to make that change?

Reply:
yup